src/main/webapp/dev/js/common/remindModal.js

```diff
@@ -2,6 +2,20 @@ import {
     sendRemindersToStudents,
 } from './instructor';
 
+function bindCheckAllCheckboxesEventHandler() {
+    const $checkallSubmitted = $('#remind-particular-checkall-submitted');
+    const $checkallNotSubmitted = $('#remind-particular-checkall-notsubmitted');
+    const $studentList = $('#studentList');
```

Comment:
Can this just be inline?

Reply:
You mean don't need to put variables for these? :D

Reply:
I meant only for the `studentList`, sorry I did not annotate the comment correctly

Reply:
Ah ok I see. Will do that after fixing merge conflicts :D

Reply:
All the best!